pkg/instancetype/instancetype.go

```diff
@@ -767,15 +812,15 @@ func (m *InstancetypeMethods) inferDefaultsFromVolumes(vm *virtv1.VirtualMachine
 		if volume.DataVolume != nil {
 			return m.inferDefaultsFromDataVolume(vm, volume.DataVolume.Name, defaultNameLabel, defaultKindLabel)
 		}
-		return "", "", fmt.Errorf("unable to infer defaults from volume %s as type is not supported", inferFromVolumeName)
+		return "", "", NewIgnoreableInferenceError(fmt.Errorf("unable to infer defaults from volume %s as type is not supported", inferFromVolumeName))
```

Comment:
What's the justification for ignoring these errors? I was assuming we'd only ignore missing labels tbh.

Reply:
There may be combinations of DataVolume/DataSource which are not valid to infer from, but virtctl can't know in advance. I intended the backend to be a bit more relaxed, so it can ignore missing labels and unsupported DV/DS combinations.